docs/platforms/cori.rst

```diff
@@ -110,10 +110,10 @@ With your nodes allocated, queue your job to start with four MPI ranks::
 
 This line launches libEnsemble with a manager and **three** workers to one
 allocated compute node, with three nodes available for the workers to launch
-user applications with the job-controller or a job-launch command.
+user applications with the executor or a job-launch command.
```

Comment:
"job-submission"?

Reply:
Probably "task-submission", but I'll need to ask @shuds13 about if Cori can launch jobs from compute nodes to know how to best reword this section.

Reply:
I would say:
user applications (via the executor or a direct run command such as mpiexec)